optuna/visualization/_terminator_improvement.py

```diff
@@ -40,6 +40,51 @@ def plot_terminator_improvement(
     error_evaluator: BaseErrorEvaluator | None = None,
     min_n_trials: int = DEFAULT_MIN_N_TRIALS,
 ) -> "go.Figure":
+    """Plot the room for future objective improvement.
```

Comment:
This visualization function differs from other visualization functions in that it takes a considerable amount of time to calculate the terminator improvement. IMO, it is better that to make a note about it somewhere. What do you think?

For example,

`Note that this function may take a counsiderable amount of time to compute terminator improvement.`